### DISCUSSION TITLE
Odroid M1S

### DISCUSSION THREAD

**Discussion by darkxst** [home-assistant/operating-system#3183] [General]
Odroid M1S now has official support and will be included in the HAOS 12.0 release. Beta images are available now on the [Release page](https://github.com/home-assistant/operating-system/releases) for testing. There is a brief install guide in the [Docs](https://github.com/home-assistant/operating-system/blob/dev/Documentation/boards/hardkernel/odroid-m1s.md).

If you have any issues with the beta builds please file them on the issues page, otherwise you can discuss here how testing goes.

**Comment by shelbycsx** [comment]
You are the man. Thanks for doing the hard work! My M1S is arriving tomorrow and I plan to try it out this weekend. Seems like the perfect platform for HA.
